packages/flutter_tools/test/general.shard/base/command_help_test.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Is this file brand new? if so it should be 2019, otherwise please ignore :)

Reply:
Yes it is new and I should probably do the same for `command_help` 😊

Reply:
Hm it actually should be 2014 according to `./dev/bots/analyze.dart` otherwise, the analysis breaks on the pipelines ... 🤔